src/main/java/frc/team4909/robot/subsystems/climber/commands/BothLiftDown.java

```diff
@@ -12,7 +13,11 @@ public BothLiftDown(){
     }
 
     public void execute() {
-        Robot.climberSubsystem.setStiltsClimbSpeed(RobotConstants.liftClimbSpeed);
+        SmartDashboard.putString("In Both Lift Down", "7");
```

Comment:
uhhhhh what is this even intended to say? @ashwinc12

Reply:
This was just for testing purpose to make sure the command was being called using the POV